template/addons/trpc/protected-question-router.ts

```diff
@@ -0,0 +1,14 @@
+import { createProtectedRouter } from "./protected-router";
+
+export const protectedQuestionRouter = createProtectedRouter().query(
```

Comment:
Kind of feel like the previous "getSession" endpoint gave more value than just a mock-query?

Reply:
Fair enough